datanode/sqlstore/positions.go

```diff
@@ -49,6 +49,14 @@ func (ps *Positions) Flush(ctx context.Context) ([]entities.Position, error) {
 	return ps.batcher.Flush(ctx, ps.Connection)
 }
 
+// AddBatch just allows you to add several entities in a single call
+func (ps *Positions) AddBatch(ctx context.Context, pos ...entities.Position) error {
+	for _, p := range pos {
+		ps.batcher.Add(p)
+	}
+	return nil
+}
+
```

Comment:
Is this used anywhere?

Reply:
Probably not, I think I added this because I'm updating 2 positions when handling the trade event, but I think I ended up just calling `Add` twice (which I suppose could just as well use variadic arguments like this)